generators/Exercises/Clock.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using Generators.Input;
+using Generators.Output;
+using Newtonsoft.Json.Linq;
+
+namespace Generators.Exercises
+{
+    public class Clock : Exercise
+    {
+        private const string paramClock1 = "clock1";
+        private const string paramClock2 = "clock2";
+        private const string paramHour = "hour";
+        private const string paramMinute = "minute";
+
+        private const string propertyCreate = "create";
+        private const string propertyEqual = "equal";
+        private const string propertyEquals = "equals";
+        private const string propertyToString = "to_string";
+
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            //canonicalData.Cases = canonicalData.Cases.Where(c => c.Property != "equal").ToArray();
+
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                if (canonicalDataCase.Property != propertyEqual)
+                {
+                    canonicalDataCase.ConstructorInput = new Dictionary<string, object>
+                    {
+                        [paramHour] = canonicalDataCase.Input[paramHour],
+                        [paramMinute] = canonicalDataCase.Input[paramMinute]
+                    };
+                }
+                else
+                {
+                    canonicalDataCase.ConstructorInput = ((JObject)canonicalDataCase.Input[paramClock1]).ToObject<Dictionary<string, object>>();
+                    canonicalDataCase.Input.Remove(paramClock1);
+
```

Comment:
This .Remove() shouldn't be needed anymore

Reply:
In this case it's still needed because I'm combined two input params `hours` and `minutes` into a singleParam so they don't automatically get removed by the `RemoveDuplicateInputEntires`.